src/DynamicObj/DynamicObj.fs

```diff
@@ -135,4 +136,129 @@ type DynamicObj internal (dict:Dictionary<string, obj>) =
         |> List.ofSeq
         |> List.sortBy (fun pair -> pair.Key)
         |> List.map (fun pair -> struct (pair.Key, pair.Value))
-        |> (fun l -> l.GetHashCode())
\ No newline at end of file
+        |> (fun l -> l.GetHashCode())
+
+    member this.ToJson () = JsonConvert.SerializeObject(this, new DynamicObjConverter())
+
+
+module DynamicObj =
+
+    let toJson (dynObj:DynamicObj) = JsonConvert.SerializeObject(dynObj, new DynamicObjConverter())
```

Comment:
please add inline XML docs (`<summary>` and `<param>` tags) This is something we should do for all public APIs in the future.